content/en/docs/started/workstation/getting-started-minikf.md

```diff
@@ -3,17 +3,8 @@ title = "MiniKF"
 description = "A fast and easy way to deploy Kubeflow on your laptop"
 weight = 35
                     
-+++
```

Comment:
Hi, @jbottum also you have to put back this line. the '+++' is needed for the title and without that the deployment will fail.